apps/galaxy/src/modules/Orders/OpenOrders/OpenOrdersList.tsx

```diff
@@ -42,16 +67,44 @@ const OpenOrdersList = () => {
     }),
   ];
 
+  const Submit = () => {
+    const Orders = checkedItems
+      .map((checked, index) => {
+        if (checked) return index;
+      })
+      .filter((checked) => checked !== undefined);
+    const orderIds = lists
+      ?.map((checkedOrders: Order, index: number) => {
```

Comment:
Since list was inferred in `useGetOrder`, we don't need to declare the type def here again.

See this doc for detail on type inference. https://www.typescriptlang.org/docs/handbook/type-inference.html